test/e2e/stack.go

```diff
@@ -27,6 +32,11 @@ type e2eCfnStack struct {
 	cfn                    *cloudformation.CloudFormation
 	iam                    *iam.IAM
 	iamRolesAnywhereCACert []byte
+	region                 string
```

Comment:
can't you get this from `e.cfn.ResolvedRegion`?

Reply:
We will need to do `*e.cfn.Config.Region`